README.md

```diff
@@ -310,6 +310,12 @@ This data source contains CVEs pertaining to RedHat Products.
 
 Access to the data is subject to [Legal Notice](https://access.redhat.com/documentation/en-us/red_hat_security_data_api/1.0/html/red_hat_security_data_api/legal-notice).
 
+### [RedHat Security Database](https://access.redhat.com/security/data) (REDHAT)
+
+This data source contains CVEs pertaining to RedHat Products.
+
+Access to the data is subject to [Legal Notice](https://access.redhat.com/documentation/en-us/red_hat_security_data_api/1.0/html/red_hat_security_data_api/legal-notice).
+
```

Comment:
```suggestion
```
This is the third copy of this paragraph! 😨 One more can be removed directly above.

Reply:
@Molkree Looks like there has been some issues with the merging of the README.